Code/Framework/AzFramework/Tests/CameraInputTests.cpp

```diff
@@ -575,4 +596,25 @@ namespace UnitTest
 
         EXPECT_THAT(removed, ::testing::IsFalse());
     }
+
+    // note: this test is attempting to mimic the behavior that happens when a user presses 'ctrl-tab' to change focus from the editor
+    // which can cause the event/update order to become irregular - this test verifies the orbit behavior does not change the position
+    // of the camera if updates are dropped (due to the editor losing focus)
+    TEST_F(CameraInputFixture, OrbitRotateCameraInputDoesNotResetPositionWithInconsitentEventAndUpdate)
+    {
+        const AzFramework::ModifierKeyStates orbitModifierKeystate = OrbitModifierKeyStates(m_orbitChannelId);
+
+        const auto cameraStartingPosition = AZ::Vector3::CreateAxisY(-20.0f);
+        m_targetCamera.m_pivot = cameraStartingPosition;
+        m_pivot = AZ::Vector3::CreateAxisY(-10.0f);
+
+        HandleEvent(AzFramework::InputState{ AzFramework::DiscreteInputEvent{ m_orbitChannelId, AzFramework::InputChannel::State::Began },

```

Comment:
note: This is a best attempt to reproduce the problem where previously multiple calls to HandleEvent (caused by strange ordering or events during Alt-Tab) to happen. The new code is now simpler and more robust and this should hopefully catch the problem if future changes are made.